docs/README.android

```diff
@@ -15,7 +15,14 @@ TOC
 
 We currently recommend Ubuntu "Xenial Xerus" (16.04) 64Bit. This is what our continuous
 integration system "jenkins" is using.
-Additionally, building from OSX Snow Leopard or Mavericks (others on your own risk ;) ) is working (see 3.1).
+
+********************** Note 
+The instructions below assumes a FRESH INSTALLATION of Ubuntu 16.04 LTS.
+OSX and Windows platforms should use a Virtualbox installation of Ubuntu.
+
+This instruction has been tested on Kodi Krypton RC-1 compiled
+for ARM architecture running on FireTV.
```

Comment:
This help file should be a general how-to and not a branch specific one. I don't care this has been tested with Krypton cause the help file should be up to date for whatever the current branch is. Devices are certainly not important because we don't care for specific devices